doc/classes/Node.xml

```diff
@@ -206,7 +206,7 @@
 			<param index="1" name="recursive" type="bool" default="true" />
 			<param index="2" name="owned" type="bool" default="true" />
 			<description>
-				Finds the first descendant of this node whose name matches [param pattern] as in [method String.match].
+				Finds the first descendant of this node whose name matches [param pattern] as in [method String.match]. Internal children are also searched over (see [code]internal[/code] parameter in [method add_child]).
```

Comment:
Turns out `find_child`/`find_children` take internal children into account unconditionally, there's no parameter for that. Not sure about wording/placement of this note.